tests/test_nimscript.nims

```diff
@@ -2,6 +2,7 @@
 # http://nim-lang.org/docs/nims.html
 
 import algorithm
+import base16
```

Comment:
Wouldn't this cause a conflict with base64 as both have procs named `encode` and `decode`? Unless you'd want people to do `from base16 import nil` and `base16.encode`, couldn't you rename them to `encodeBase16` and `decodeBase16`

Reply:
No, its `base16.encode` or `base64.encode` if both modules are imported.